.travis.yml

```diff
@@ -52,4 +52,4 @@ script:
 
 after_success:
     - source ./scripts/travis/script_deploy_android_snapshot.sh
-    - source ./scripts/travis/script_deploy_android_demo.sh
+    #- source ./scripts/travis/script_deploy_android_demo.sh
```

Comment:
Just a note, if this was intended to be commented, and if so, does it make sense to take it out completely? I am neutral to it. Also if we decide to take the demo deployment out, maybe remove the shell script also.

Reply:
Yeah I did comment this intentionally in a7d94f3

I expect to re-enable this later, but it shouldn't block the other changes.

Reply:
Cool, just wanted to clarify.